src/SDKs/Authorization/Authorization.Tests/Graph/GraphManagementClient.cs

```diff
@@ -248,11 +248,12 @@ private HttpRequestMessage CreateRequest(string uri, HttpMethod method, string b
 
             if (this.testEnvironment != null && this.testEnvironment.TokenInfo != null)
             {
-                // Not Supported in current code
-                // var tokenCredentials = (TokenCredentials)this.testEnvironment.Credentials;
-                // httpRequest.Headers.Authorization = new AuthenticationHeaderValue(
-                //                                this.testEnvironment.AccessTokenType,
-                //                                this.testEnvironment.AuthorizationContext.AccessToken);
+				// Not Supported in current code
+				// var tokenCredentials = (TokenCredentials)this.testEnvironment.Credentials;
+				httpRequest.Headers.Authorization = new AuthenticationHeaderValue("Bearer",
+					"Put the access token for graph here for the tests to work in record mode");
```

Comment:
i think this is for cases where service principal login doesn't have sufficient privileges to perform delete operations against the tenant. 

---
In reply to: [118618063](https://github.com/Azure/azure-sdk-for-net/pull/3278#discussion_r118618063) [](ancestors = 118618063)